### PR TITLE
Remove hidden property

### LINKED ISSUE #281: [Simulation] Remove `hidden` property in favour of `edited`.
In the simulation view _parameters_ and _variables_ have two different properties `hidden` and `edited` who behaves differently.

1. **`hidden` needs to be removed, as it's out of scope for now.**

2. **`edited` needs to be kept, and be renamed `hidden`.**
   This part is more delicate has `edited` is a positive test, and `hidden` is a negative one.

### PR DESCRIPTION
**Pull request by emilykuang**
**What**

- fix #281 by removing the `hidden` property of parameters and variables and renaming `edited` property to `displayed`
- `displayed` makes more sense than `hidden` since it was the opposite meaning

**Testing**

- Select a model and open the simulation view 
- Add parameters using the plus sign and make sure the yellow dots appear in the Model Panel
- Remove parameters using the no icon and make sure the yellow dots disappear 

Screenshot:
image

Updated screenshot with highlight box:
image

### CODE REVIEW

File: client/src/views/Simulation/components/ModelPanel.vue [hunk 1]
```diff
@@ -15,7 +15,7 @@
     <global-graph
       v-if="graph"
       :data="graph"
-      :edited-nodes="editedNodes"
+      :edited-nodes="displayedNodes"
```
Comment (YohannParis) [inline client/src/views/Simulation/components/ModelPanel.vue:18]: you might change the name of the prop as well from `edited-nodes` to `displayed-nodes` on the `<global-graph>` component.

**Review by YohannParis — APPROVED** [review]
Looks good!

File: client/src/views/Simulation/components/ModelPanel.vue (hunk 1 above)
Comment (emilykuang) [reply client/src/views/Simulation/components/ModelPanel.vue]: Thanks for the suggestion, updated.

**Review by YohannParis — APPROVED** [review]

**Comment by RosaRomeroGomez** [comment]
The yellow dots are not disappearing for me when I unselect one of the parameters or variables from the Model panel. I suspect it is related to the bug you logged here: https://github.com/uncharted-aske/HMI/issues/328 Are you guys experiencing the same issue? @YohannParis @mj3cheun ?

**Review by RosaRomeroGomez — APPROVED** [review]

**Comment by RosaRomeroGomez** [comment]
> > The yellow dots are not disappearing for me when I unselect one of the parameters or variables from the Model panel. I suspect it is related to the bug you logged here: #328 Are you guys experiencing the same issue? @YohannParis @mj3cheun ?
> 
> Yes, I noticed this issue too before I started refactoring. The dots appear and disappear as expected when you only use the buttons at the top or only by double-clicking on the nodes. However, if you try to add all dots using the + button, then try to remove one by double-clicking, the dot doesn't disappear.

Thanks @emilykuang for the clarifications. I think this PR seems fine. Maybe you can assign yourself that bug and see if you can find the issue?
